### DISCUSSION TITLE
Add an option to add a VTT SRT subtitle file and bypass transcription/translation

### DISCUSSION THREAD

**Discussion by ROBERT-MCDOWELL** [KoljaB/TurnVoice#10] [Ideas]
keep the audio analyzing for diarization and voice cloning, then use directly the VTT/SRT for TTS with timestamp sync.
no need to be realtime, a background process even working with CPU would be great.
This feafure can be very useful for blind people who'd like to listen videos translated only with subtitles or for busy people not watching the screen but would llke to understand specific videos/audio.

**Comment by KoljaB** [comment]
That would be a great feature. Maybe if I find time I will implement some stuff into turnvoice again some day. Would also love to take a PR from the community.

I don't maintain turnvoice currently, you can see this from the last update. Let me explain: this project was done in few days of hardcore coding. My creative process just works like that: with intense, short bursts of work to bring ideas to life. Now complex projects like these would need care only to keep up the status quo. 

Unfortunately in the python world often installations break over time due to changing dependencies. So it forces you to pay constant attention to keep your project up. I can not do that for every of my projects (I do regular maintenance for RealtimeTTS, RealtimeSTT and Linguflex and I can barely keep up because these already require a huge amount of my time).

**Comment by ROBERT-MCDOWELL** [comment]
don't worry I'm often the same as you, that's why I like your profile picture and what's written inside :)
If I can at least install turnvoice on Linux maybe I can give some time to maintain and add this precious feature.
I saw some dev guys with the help of A.I. created a script that check automatically package versions and fixes...
If you have a little time to implement the VTT/SRT option so I will have more time for test and fixes too.
